src/Components/ModalCategory/ModalCategory.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import './ModalCategory.scss';
+
+class ModalCategory extends React.Component {
+  render() {
+    const { currentMenu, onClick } = this.props;
+
+    return (
+      <div className="ModalCategory">
+        {currentMenu === 2 && <CollectionDropdown onClick={onClick} />}
+        {currentMenu === 3 && <CategoryDropdown onClick={onClick} />}
+      </div>
+    );
+  }
+}
+
+class CollectionDropdown extends React.Component {
+  render() {
+    const imgSrc =
+      'https://www.chanel.com/images/q_auto,f_auto,fl_lossy,dpr_auto/w_450/FSH-1591614379103-hv4desktop.jpg';
+    return (
+      <div className="CollectionDropdown">
+        <div className="imgWrapper">
+          <img src={imgSrc} alt="collection" />
```

Comment:
지금처럼 imgSrc 에 들어가야 하는 이미지 링크가 변하는 상황이 아니라면 굳이 render문 안에 넣어서 매 render 때마다 초기화 시킬 필요가 없어보입니다. 아래 보시는 것 처럼 직접 넣어주시거나 아예 이런 것들만 모아놓는 js 파일을 따로 관리해서 import 해서 넣어주시는 방법도 있습니다.
```suggestion
  render() {
    return (
      <div className="CollectionDropdown">
        <div className="imgWrapper">
          <img src="https://www.chanel.com/images/q_auto,f_auto,fl_lossy,dpr_auto/w_450/FSH-1591614379103-hv4desktop.jpg" alt="collection" />
```